interface/reports/daily_summary_report.php

```diff
@@ -0,0 +1,409 @@
+                                        <td><?php echo isset($information['visits']) ? $information['visits'] : 0; ?></td>
+                                        <td align="right"><?php echo isset($information['payments']) ? number_format($information['payments'], 2) : number_format(0, 2); ?></td>
+                                        <td align="right"><?php echo isset($information['paidAmount']) ? number_format($information['paidAmount'], 2) : number_format(0, 2); ?></td>
+                                        <td align="right">
+                                            <?php
+                                            if (isset($information['payments']) || isset($information['paidAmount'])) {
+                                                $dueAmount = number_format(str_replace(",", "", $information['payments']) - str_replace(",", "", $information['paidAmount']), 2);
+                                            } else {
+                                                $dueAmount = number_format(0, 2);
+                                            }
+                                            echo $dueAmount;
+                                            ?>
+                                        </td>
+                                    </tr>
+                                    <?php
+                                    if (count($dailySummaryReport) > 0) { // calculate the total count of the appointments, new patient,visits, payments, paid amount and due amount
+                                        $totalAppointments += $information['appointments'];
+                                        $totalNewRegisterPatient += $information['newPatient'];
+                                        $totalVisits += $information['visits'];
+                                        $totalPayments += str_replace(",", "", $information['payments']);
+                                        $totalPaidAmount += str_replace(",", "", $information['paidAmount']);
+                                        $totalDueAmount += $dueAmount;
+                                    }
+                                }
+                            }
+                        }
+                    }
+                    ?>
+                    <!--display total count-->
+                    <tr class="totalrow">
+                        <td>Total</td>
+                        <td>-</td>
+                        <td>-</td>
+                        <td><?php echo $totalAppointments; ?></td>
+                        <td><?php echo $totalNewRegisterPatient; ?></td>
+                        <td><?php echo $totalVisits; ?></td>
+                        <td align="right"><?php echo number_format($totalPayments, 2); ?></td>
+                        <td align="right"><?php echo number_format($totalPaidAmount, 2); ?></td>
+                        <td align="right"><?php echo number_format($totalDueAmount, 2); ?></td>
```

Comment:
We have used text() function as per your above comments.
Please see the changes at pull request https://github.com/bradymiller/openemr/pull/9